tests/utils/SimplifyTest.cpp

```diff
@@ -106,13 +116,15 @@ class SimplifyTest: public testing::Test
  */
 TEST_F(SimplifyTest, CircleMaxResolution)
 {
-    simplifier.max_deviation = 999999; //For this test, the maximum deviation should not be an issue.
+    simplifier.max_deviation = 999999; // For this test, the maximum deviation should not be an issue.
     circle = simplifier.polygon(circle);
 
-    for(size_t point_index = 1; point_index + 1 < circle.size(); point_index++) //Don't check the last vertex. Due to odd-numbered vertices it has to be shorter than the minimum.
+    for (size_t point_index = 1; point_index + 1 < circle.size();
+         point_index++) // Don't check the last vertex. Due to odd-numbered vertices it has to be shorter than the minimum.
```

Comment:
I'd rather put the comment on a separate line than the incrementer (?) of the for loop.

Reply:
Agreed gonna check how to add this rule to clang-format